.github/workflows/DEVELOP-CI-CD.yml

```diff
@@ -0,0 +1,63 @@
+name: DEVELOP-CI-CD
+
+on:
+    push:
+        branches:
+            - develop
+
+jobs:
+    install-dependencies:
+        runs-on: ubuntu-latest
+        container: node:21-alpine
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: install dependencies
+              run: yarn install --frozen-lockfile
+
+            - name: Tar node_modules directory
+              run: tar czf node_modules.tar.gz -C node_modules .
+
+            - name: Upload node_modules_artifact
+              uses: actions/upload-artifact@v4
+              with:
+                  name: node_modules
+                  path: node_modules.tar.gz
+
+    release:
+        needs: [run-unit-tests]
+        runs-on: ubuntu-latest
+
+        env:
+            image_tag: ${{vars.DOCKER_IMAGE_OWNER}}/kiosk-auth-dev:${GITHUB_SHA}
+            image_latest: ${{vars.DOCKER_IMAGE_OWNER}}/kiosk-auth-dev:latest
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Login to Docker Hub
+              uses: docker/login-action@v3
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+            - name: Build Docker Images
+              run: |
+                  docker build -t ${{ env.image_tag }} .
+                  docker tag ${{ env.image_tag }} ${{ env.image_latest }}
+
+            - name: Push Docker Images
+              run: |
+                  docker push ${{ env.image_tag }}
+                  docker push ${{ env.image_latest }}
+
+    deploy:
+        needs: [release]
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Trigger deploy
+              run: curl --request POST '${{ secrets.DEPLOY_AUTH_DEVELOP_WEBHOOK }}'
```

Comment:
remove the deploy job since we are not deploying our apps on render anymore and it is not needed